lib/realtime/tenants/connect/create_partitions.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Realtime.Tenants.Connect.CreatePartitions do
```

Comment:
Maybe okay for the short term, but I guarantee we'll get folks trying to create messages here when partitions don't exist because the creation of these depend on a client connection.